package.json

```diff
@@ -106,36 +106,36 @@
   "peerDependencies": {
     "@dialpad/dialtone-emojis": "workspace:*",
     "@dialpad/dialtone-icons": "workspace:*",
-    "@linusborg/vue-simple-portal": "^0.1.5",
-    "@tiptap/core": "^2.0.3",
-    "@tiptap/extension-blockquote": "^2.1.16",
-    "@tiptap/extension-bold": "^2.1.16",
-    "@tiptap/extension-bullet-list": "^2.1.16",
-    "@tiptap/extension-code-block": "^2.0.3",
-    "@tiptap/extension-document": "^2.0.3",
-    "@tiptap/extension-hard-break": "^2.0.3",
-    "@tiptap/extension-italic": "^2.1.16",
-    "@tiptap/extension-link": "^2.1.16",
-    "@tiptap/extension-list-item": "^2.1.16",
-    "@tiptap/extension-mention": "^2.1.11",
-    "@tiptap/extension-ordered-list": "^2.2.0",
-    "@tiptap/extension-paragraph": "^2.0.3",
-    "@tiptap/extension-placeholder": "^2.0.3",
-    "@tiptap/extension-strike": "^2.1.16",
-    "@tiptap/extension-text": "^2.0.3",
-    "@tiptap/extension-text-align": "^2.2.0",
-    "@tiptap/extension-underline": "^2.1.16",
-    "@tiptap/pm": "^2.0.3",
-    "@tiptap/suggestion": "^2.1.4",
-    "@tiptap/vue-2": "^2.0.3",
-    "@tiptap/vue-3": "^2.0.3",
-    "date-fns": "^2.30.0",
-    "docopt": "^0.6.2",
-    "emoji-regex": "^10.2.1",
-    "emoji-toolkit": "^8.0.0",
-    "requireindex": "^1.2.0",
+    "@linusborg/vue-simple-portal": "0.1.5",
+    "@tiptap/core": "2.0.3",
+    "@tiptap/extension-blockquote": "2.1.16",
+    "@tiptap/extension-bold": "2.1.16",
+    "@tiptap/extension-bullet-list": "2.1.16",
+    "@tiptap/extension-code-block": "2.0.3",
+    "@tiptap/extension-document": "2.0.3",
+    "@tiptap/extension-hard-break": "2.0.3",
+    "@tiptap/extension-italic": "2.1.16",
+    "@tiptap/extension-link": "2.1.16",
+    "@tiptap/extension-list-item": "2.1.16",
+    "@tiptap/extension-mention": "2.1.11",
+    "@tiptap/extension-ordered-list": "2.2.0",
+    "@tiptap/extension-paragraph": "2.0.3",
+    "@tiptap/extension-placeholder": "2.0.3",
+    "@tiptap/extension-strike": "2.1.16",
+    "@tiptap/extension-text": "2.0.3",
+    "@tiptap/extension-text-align": "2.2.0",
+    "@tiptap/extension-underline": "2.1.16",
+    "@tiptap/pm": "2.0.3",
+    "@tiptap/suggestion": "2.1.4",
+    "@tiptap/vue-2": "2.0.3",
+    "@tiptap/vue-3": "2.0.3",
+    "date-fns": "2.30.0",
+    "docopt": "0.6.2",
+    "emoji-regex": "10.2.1",
+    "emoji-toolkit": "8.0.0",
+    "requireindex": "1.2.0",
     "stylelint": "15.11.0",
-    "tippy.js": "^6.3.7"
+    "tippy.js": "6.3.7"
```

Comment:
Can we set `save-prefix=""` in our npmrc so it always installs exact versions by default when we do `pnpm add`?

Reply:
Didn't knew about that, let me check.